cryostatio/cryostat-web
# feat(dashboard): full view dashboard card

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Fixes: #739 

## Description of the change:

Added a subroute to view the card in "full-screen" mode. This approach is similar to Grafana Dashboard. The route requires a query paramter `cardId` to find the card configuration. If none or invalid, an empty state will be shown.

## Motivation for the change:

See #739 

## How to test

1. Create any dashboard card.
2. Click on top-right kebab icon to show action.
3. Click `View` to see full mode.
4. Notice the breadcrumb has the card Id (same as query param) and allows going back to main dashboard.
